docs/src/rules/no-mixed-spaces-and-tabs.md

```diff
@@ -23,10 +23,15 @@ Examples of **incorrect** code for this rule:
 /*eslint no-mixed-spaces-and-tabs: "error"*/
 
 function add(x, y) {
+// --->..return x + y;
```

Comment:
I don't think this is very clear. I'm guessing that you're trying to show where the tabs and spaces are, but it's just confusing to me. I'd leave these out.

Reply:
The comments were already there before my changes. No problem to re-remove them if that's fine for @mdjermanovic.

Reply:
I'm fine with removing these comments.